CppUTest.dsp

```diff
@@ -90,71 +90,71 @@ SOURCE=.\src\CppUTestExt\CodeMemoryReportFormatter.cpp
 # End Source File
 # Begin Source File
 
-SOURCE=.\src\CppUTestExt\MemoryReportAllocator.cpp
```

Comment:
The changes in the .dsp are a bit annoying. Would have to be edited manually. Essentually, we add one file to AllTests. That's all.